src/js/components/form-elements/FieldArrayComponent.jsx

```diff
@@ -54,6 +54,8 @@ class FieldArrayComponent extends Component {
     const { maxTableHeight = 'calc(100vh - 400px)', virtualized } = fieldsConfig;
     const addRow = (row = {}, index = null) => {
       if (index === null) {
+        const table = document.querySelectorAll('[role="rowgroup"]')[0];
```

Comment:
Why does it work without this part on the inbound sm?

Reply:
@awalkowiak It works on inbound, outbound and for requests because FieldArrayComponent is used on each of those workflows.